build-tools/automation/azure-pipelines.yaml

```diff
@@ -231,3 +232,204 @@ stages:
         artifactName: win-build-test-results
         targetPath: $(Build.ArtifactStagingDirectory)
       condition: always()
+
+- stage: test
+  displayName: Test
+  dependsOn: mac_build
+  jobs:
+  - job: mac_apk_tests
```

Comment:
This will be duplicative, but shouldn't we also have a `windows_vsix_tests` job as a "peer" to this job, but likewise depending on `mac_build` so that we can test the macOS-generated `.vsix` on Windows?  (This is arguably duplicative of the existing `win_build_test`, except `win_build_test` tests the artifacts built as part of that job, which is *not* the installer we'll *actually* redistribute.)

Granted, we might not be able to do that yet, but presumably at some point we should...

Reply:
Agreed, we should look to add a Windows test stage which runs our desktop unit tests against our installers in the future.